docs/field-values.asciidoc

```diff
@@ -314,6 +326,18 @@ The session category is applied to events and metrics regarding logical persiste
 start, end, info
 
 
+[float]
+[[ecs-event-category-threat]]
+==== threat
+
+Relating to cybersecurity threats. Use this category to visualize and analyze events describing threat actors' targets, motives, or behaviors.
```

Comment:
```suggestion
Use this category to visualize and analyze events describing threat actors' targets, motives, or behaviors.
```

Reply:
I think that `threat` could be used to describe more than just cybersecurity. Especially if this is expanded at some point to include physical threat uses - such as for traditional kinetic threats or human threats like you could observe in law enforcement, terrorism, or economic espionage.

Reply:
Great points - will make this adjustment.